tests/host_tools/memory.py

```diff
@@ -65,13 +72,10 @@ def _memory_cop(mem_size_mib, pid):
             mem_total += rss
 
         if mem_total > MAX_MEMORY:
-            print('ERROR! Memory usage exceeded limit: {}'
-                  .format(mem_total))
-            exit(-1)
+            exceeded_queue.put(mem_total)
```

Comment:
With this change I think the doc string for the function needs to be updated as well.